### ISSUE TITLE
getting issue with "ip -4 -j address show interface" command in  interfaces.py.

### ISSUE THREAD

**Issue by smruti77** [avocado-framework/avocado#3674]
in few machine getting output with empty dictionary so tests are failing because of this.
for example:

 ip -4 -j address show enP72p1s0
[{},{},{"ifindex":9,"ifname":"enP72p1s0","flags":["BROADCAST","MULTICAST","UP","LOWER_UP"],"mtu":1500,"qdisc":"mq","operstate":"UP","group":"default","txqlen":1000,"addr_info":[{"family":"inet","local":"13.13.13.13","prefixlen":24,"broadcast":"13.13.13.255","scope":"global","noprefixroute":true,"label":"enP72p1s0","valid_life_time":4294967295,"preferred_life_time":4294967295}]},{}]

**Comment by manvanthar** [comment]
hi @beraldoleal,  is this something  you could help with the new network util changes?

Currently its blocking our tests.

**Comment by smruti77** [comment]
@beraldoleal another thing in older distro's "-j" options is not supporting their what we need do.

**Comment by beraldoleal** [comment]
Hi @msruti77 and @manvanthar.

Can you please send the error message (traceback) that you are getting?

**Comment by beraldoleal** [comment]
About the old versions without `-j` support, this was discussed with Cleber and during our meetings:  Avocado #69 is supporting RHEL 7. `-j` it was introduced in Avocado #77. For this version, we support RHEL 8 and beyond (and new fedora/centos versions).

What is the distro that you are running?

**Comment by smruti77** [comment]
@beraldoleal ,"ERROR: argument of type 'NoneType' is not iterable" this is the error what i am getting .its because of ,
we are taking output of "ip -4 -j address show enP72p1s0" with [0].
```
try:
            details = self._get_interface_details(version)
            addr_info = details[0].get('addr_info')
```
but in few machine we are getting output with {} so its failing.
```'ip -4 -j address show enP72p1s0'
[{},{},{"ifindex":9,"ifname":"enP72p1s0","flags":["BROADCAST","MULTICAST","UP","LOWER_UP"],"mtu":1500,"qdisc":"mq","operstate":"UP","group":"default","txqlen":1000,"addr_info":[{"family":"inet","local":"13.13.13.13","prefixlen":24,"scope":"global","label":"enP72p1s0","valid_life_time":4294967295,"preferred_life_time":4294967295}]},{}]
```

**Comment by smruti77** [comment]
> About the old versions without `-j` support, this was discussed with Cleber and during our meetings: Avocado #69 is supporting RHEL 7. `-j` it was introduced in Avocado #77. For this version, we support RHEL 8 and beyond (and new fedora/centos versions).
> 
> What is the distro that you are running?

@beraldoleal i tried in RHEL 7.7, their it is not supporting.

**Comment by beraldoleal** [comment]
@smruti77 about the error on different output format, please see if this PR will fix the problem: #3675

**Comment by beraldoleal** [comment]
> @beraldoleal i tried in RHEL 7.7, their it is not supporting.

Yes, afaik the new features introduced on  Avocado > # 69 are for RHEL 8 and beyond.

**Comment by smruti77** [comment]
> @smruti77 about the error on different output format, please see if this PR will fix the problem: #3675

@beraldoleal thanks,that is working fine.
